files/usr/share/functions/device_functions.sh

```diff
@@ -3,6 +3,7 @@
 [ -e /usr/share/functions/custom_device.sh ] && . /usr/share/functions/custom_device.sh
 . /lib/functions.sh
 . /lib/functions/leds.sh
+. /usr/share/libubox/jshn.sh
```

Comment:
Precisa carregar a jshn aqui, @danielatk ? É por causa do customo_device? Se for isso, não é melhor carregar direto no arquivo? Só uma dúvida mesmo....

Reply:
> Precisa carregar a jshn aqui, @danielatk ? É por causa do customo_device? Se for isso, não é melhor carregar direto no arquivo? Só uma dúvida mesmo....

Não precisa. Isso é resquício de quando eu tinha feito uma função para settar vlan usando informação do custom_switch_port diretamente no flashbox (ao invés da solução atual de pegar isso pelo flashman).